features/step_definitions/covering_letter_steps.rb

```diff
@@ -3,17 +3,5 @@
 end
 
 Then(/^it should include a covering letter$/) do
-  output_document = OutputDocument.new(@appointment_summary)
-  expected = {
-    attendee_address_line_1: output_document.attendee_address_line_1,
-    attendee_address_line_2: output_document.attendee_address_line_2,
-    attendee_address_line_3: output_document.attendee_address_line_3,
-    attendee_town: output_document.attendee_town,
-    attendee_county: output_document.attendee_county,
-    attendee_postcode: output_document.attendee_postcode,
-    attendee_name: output_document.attendee_name,
-    lead: output_document.lead
-  }
-
-  expect_uploaded_csv_to_include(expected)
+  # behaviour is outside of the system, nothing to assert
```

Comment:
I misunderstood this when we were discussing in person – the original assertion is still valid, we need to make sure the CSV contains all details needed to render a covering letter.